Scenario/ScenarioConstellation.py

```diff
@@ -0,0 +1,438 @@
+        logging.info("Start defining Plan...")
+        self.define_plan()
+        logging.info("Finish defining Plan...")
+
+    def execute(self):
+        """ Execute the scenario until the fleet converges using a method from the fleet class.
+        """
+        # self.fleet.design(self.plan, self.clients, verbose=verbose)
+        logging.info("Start executing...")
+        try:
+            self.fleet.design_constellation(self.plan, verbose=False)
+            logging.info("Finish executing...")
+            return True
+        except RuntimeWarning as warning:
+            return warning
+
+    def define_constellation(self):
+        """ Define constellation object.
+        """
+        # Define relevant orbits
+        logging.info("Gathering satellite orbits...")
+        self.define_constellation_orbits()
+
+        # Check if satellites volume is known, otherwise an estimate is provided
+        if int(self.sat_volume.value) == 0:
+            logging.info("Estimating satellite volume...")
+            self.estimate_satellite_volume()
+
+        # Define a reference satellite
+        logging.info("Generating the reference satellite...")
+        self.reference_satellite = Satellite('Reference_' + self.constellation_name + '_satellite',
+                                             self.sat_mass,
+                                             self.sat_volume,
+                                             self.sat_insertion_orbit,
+                                             self.sat_operational_orbit,
+                                             state='standby',
+                                             is_stackable=False)
+
+        # Instanciate ConstellationSatellites object
+        logging.info("Instanciating the constellation...")
+        self.constellation = Constellation(self.constellation_name)
+
+        logging.info("Populating the constellation based on the reference satellite...")
+        self.constellation.populate_standard_constellation(self.constellation_name,
+                                                           self.reference_satellite,
+                                                           number_of_planes=self.n_planes,
+                                                           sat_per_plane=self.n_sats_per_plane,
+                                                           plane_distribution_angle=self.plane_distribution_angle,
+                                                           altitude_offset=0 * u.km)
+
+        # Log satellites distribution
+        for _, satellite in self.constellation.satellites.items():
+            logging.info(f"Sat {satellite.ID} has {satellite.insertion_orbit}, {satellite.insertion_orbit.raan} RAAN, {satellite.insertion_orbit.nu} nu orbit")
+
+        # Plot if verbose
+        if self.verbose:
+            logging.info("Start plotting Clients...")
+            self.satellites.plot_3D_distribution(save="3D_plot", save_folder=self.data_path)
```

Comment:
@Laubeee Satellites is undefined here. If I change to `self.constellation.satellites..plot_3D_distribution(save="3D_plot", save_folder=self.data_path)` I get the following error from astropy:

```python
Traceback (most recent call last):
  File "Contents/plugins/python/helpers/pydev/pydevd.py", line 1491, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "Contents/plugins/python/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "tcat/Run_Constellation.py", line 59, in <module>
    scenario.setup()
  File "tcat/Scenario/ScenarioConstellation.py", line 116, in setup
    self.define_constellation()
  File "tcat/Scenario/ScenarioConstellation.py", line 186, in define_constellation
    self.constellation.satellites.plot_3D_distribution(save="3D_plot", save_folder=self.data_path)
AttributeError: 'dict' object has no attribute 'plot_3D_distribution'
```